### PR TITLE
RenderTemplate request additional parameters

### PR DESCRIPTION
**Pull request by serdyuka**

### CODE REVIEW

**Comment by serdyuka** [comment]
@makasim review please

File: src/Payum/Core/Request/RenderTemplate.php [hunk 1]
```diff
@@ -60,4 +60,13 @@ public function setResult($result)
     {
         $this->result = $result;
     }
+
+    /**
+     * @param string $name
+     * @param mixed  $value
+     */
+    public function addParameter($name, $value)
```
Comment (makasim) [inline src/Payum/Core/Request/RenderTemplate.php]: let's add one more merhod `setParamter`. and this one must throw exception if there is already such parameter set where `setParameter` should always set\overwrite it. 

Could you rename context to `parameters` and gettter too.

File: src/Payum/Stripe/Resources/views/Action/obtain_js_token.html.twig [hunk 2]
```diff
@@ -5,7 +5,7 @@
 {% block payum_body %}
     {{ parent() }}
 
-    <form action="" method="POST" id="payment-form">
+    <form action="{{ captureActionUrl|default('') }}" method="POST" id="payment-form">
```
Comment (makasim) [inline src/Payum/Stripe/Resources/views/Action/obtain_js_token.html.twig]: name it `actionUrl`

**Comment by serdyuka** [comment]
@makasim done

**Comment by makasim** [comment]
Thanks

**Pull request merged**